content/docs/reference/qri_python_commands.md

```diff
@@ -0,0 +1,49 @@
+---
+metaTitle: "Qri Python Commands"
+metaDescription: "An Overview of Qri Python Client Commands"
+weight: 9
```

Comment:
I went ahead and ordered this after our other pages unless we prefer this to be featured at the top of the reference section?